### ISSUE TITLE
ReferenceError: buidlerArguments is not defined

### ISSUE THREAD

**Issue by okwme** [MolochVentures/moloch#98]
trying a fresh deploy from master on gitpod, following instructions i get this error:
```
ReferenceError: buidlerArguments is not defined
```
Not sure if it's because it needs to be updated with hardhat, the new name for buidler...

**Comment by santisiri** [comment]
Yes, very likely that's a deprecated `buidler` call and should be updated with the equivalent in `hardhat`

**Comment by alcuadrado** [comment]
Yes, `buidlerArguments` is now `hardhatArguments`.

**Comment by santisiri** [comment]
i'll try to find some time and make a new pull to finish updating this repo

**Comment by kh0rvus** [comment]
just submitted a pr to fix this!

**Comment by rvmelkonian** [comment]
anyone know a workaround for this issue in the time being? I'm trying to deploy locally and build some new frontends for the Moloch

**Comment by 0xmovses** [comment]
A work around is to find all `builderArguments` and replace with `hardhatArguments`
